source/error_handler.cc

```diff
@@ -33,11 +33,17 @@ ErrorHandler<ntables>::ErrorHandler (   const std::string name,
 template <int ntables>
 void ErrorHandler<ntables>::declare_parameters (ParameterHandler &prm)
 {
+  std::string dummy_names;
+  for (unsigned int i=0; i<ntables-1; ++i)
+    {
+      dummy_names = dummy_names + "error ,";
```

Comment:
can you make it such that the name is not the same, and that it only adds names for i>1: 
~~~
dummy_names = "error";
for(unsigned int i=1; i<ntables-1; ++i)
    dummy_names += ", error"+Utilities::int_to_string(i);